docs: update compute addons with pg repl slots and wal senders (#18150)
supabase/supabase: apps/docs/pages/guides/realtime/bring-your-own-database.mdx

> - `max_replication_slots`: we recommend `10` because Realtime requires a few slots plus the slots you'll need for your non-Realtime logical replication needs.

Reviewer: Should we change this recommendation of `10` based on the new info?

Reviewer 2: It's fine to leave at 10 for external databases I think (though really it should delve into `existing replication slots + margin for realtime + margin for your growth` or similar)

- `max_slot_wal_keep_size`: we recommend `1024` (MB) so Realtime can attempt to deliver more database changes stored in Postgres.

> On [Compute Add-ons](/docs/guides/platform/compute-add-ons#postgres-replication-slots-and-wal-senders) page you can find the `max_replication_slots` configurations we use on our cloud offering, with recommended values based on instance size.

Reviewer: ```suggestion
See [Compute Add-ons](/docs/guides/platform/compute-add-ons#postgres-replication-slots-and-wal-senders) for the recommended `max_replication_slots` at different instance sizes based on the values we use for our own cloud offering.
```
